exercises/practice/ordinal-number/OrdinalNumber.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/*
+This is only a SKELETON file for the "Ordinal number" exercise. It's been provided as a
+convenience to get you started writing code faster.
+
+Remove this comment before submitting your exercise.
+ */
+
+function ordinalNumber(int $number): string
```

Comment:
What do you think about giving this function an action-oriented name?

```suggestion
function toOrdinal(int $number): string
```

Reply:
Good point. Fixed.